src/uwtools/drivers/support.py

```diff
@@ -14,6 +17,22 @@ def graph() -> str:
     return _iotaa.graph()
 
 
+def set_driver_docstring(driver_class: Type) -> None:
+    """
+    Appends inherited parameter descriptions to the driver's own docstring.
+
+    :param driver_class: The class whose docstring to update.
+    """
+    head_old = driver_class.__doc__
+    body_old = driver_class.__mro__[1].__doc__
```

Comment:
`__mro__` is the "method resolution order" list: An ordered list of classes to consult to look for a method name. The class at index 0 is the current class, and index 1 is the parent class. The parent class, in this case, has the docstring with the parameter descriptions needed here.

Reply:
Would it be clearer to name this thing that many folks may never have heard of?

```
parent_class = driver_class.__mro__[1]
body_old = parent_class.__doc__
```